project/src/components/app/app.tsx

```diff
@@ -1,14 +1,35 @@
-import MainPage from '../main-page/main-page';
+import { BrowserRouter, Navigate, Route, Routes } from 'react-router-dom';
+import { AppRoute } from '../../const';
+import Favorites from '../../pages/favorites/favorites';
+import Login from '../../pages/login/login';
+import Main from '../../pages/main/main';
+import NotFound from '../../pages/not-found/not-found';
+import Room from '../../pages/room/room';
+import Layout from '../layout/layout';
+import PrivateRoute from '../private-route/private-route';
 
 type AppProps = {
-  loggedIn: boolean,
+  authorizationStatus: string,
```

Comment:
```suggestion
  authorizationStatus: AuthorizationStatus,
```
это не просто строка) А типизированая строка 😃

Reply:
поменяю